RecoEgamma/ElectronIdentification/src/ElectronMVAEstimatorRun2.cc

```diff
@@ -104,6 +104,9 @@ float ElectronMVAEstimatorRun2::
 mvaValue( const edm::Ptr<reco::Candidate>& candPtr, const edm::EventBase & iEvent) const {
 
   const int iCategory = findCategory( candPtr );
+
+  if (iCategory < 0) return 0;
```

Comment:
Normally, in case of bad  inputs a meaningless mva value is returned (e.g. -9999)
Here you return a 0: is it coherent with the logic of the algorithm?